amps/ags/rm-community/rm-community-rest-api-explorer/src/main/webapp/definitions/gs-core-api.yaml

```diff
@@ -2498,36 +2498,38 @@ paths:
           schema:
             $ref: '#/definitions/Error'
   ##retention-schedule
-  '/record-categories/{recordCategoryId}/retention-schedules':
+  '/record-categories/{recordCategoryId}/retention-schedule':
```

Comment:
This is a good spot, but I would leave this as "schedules" even though we know there can only be one. It will be consistent with e.g. `file-plans` endpoint.

Reply:
done the changes as per comment.